node/src/chain_spec.rs

```diff
@@ -565,30 +568,31 @@ fn amplitude_genesis(
 		oracle: amplitude_runtime::OracleConfig {
 			max_delay: u32::MAX,
 			oracle_keys: vec![
+				Key::ExchangeRate(CurrencyId::Native),
 				Key::ExchangeRate(CurrencyId::XCM(0)),
-				Key::ExchangeRate(MAINNET_USDC_CURRENCY_ID),
+				Key::ExchangeRate(TESTNET_USDC_CURRENCY_ID),
```

Comment:
Nit: I changed this for testing and forgot to change it back
```suggestion
				Key::ExchangeRate(MAINNET_USDC_CURRENCY_ID),
```